Native-Frameworks/Views/AlertView/PopupSelectionView.swift

```diff
@@ -4,49 +4,58 @@ struct PopupSelectionView: View {
     
     @Binding var isGridviewSelected: Bool
     @Binding var isListviewSelected: Bool
+    @Environment(\.presentationMode) var presentation
```

Comment:
@jantimar can you suggest a possible solution when tapping either `isListviewSelected` or `isGridviewSelected` I can dismiss the `PopupSelectionView`?